Microsoft.Toolkit.Uwp.SampleApp/Package.appxmanifest

```diff
@@ -36,6 +36,16 @@
             <Task Type="timer" />
           </BackgroundTasks>
         </Extension>
+        <uap:Extension Category="windows.fileTypeAssociation">
+          <uap:FileTypeAssociation Name="textname">
+            <uap:SupportedFileTypes>
+              <uap:FileType>.txt</uap:FileType>
+              <uap:FileType>.log</uap:FileType>
+              <uap:FileType>.md</uap:FileType>
```

Comment:
Why are we explicitly registering to file types to open with the sample app?

Reply:
I needed to register some filetypes so that the file open dialog can show some valid types when it is invoked by the sample app. I thought I needed to do this. Let me check if everything works without this.

Reply:
You are right. This wasn't necessary. I have removed these lines. Thanks.